frontend/src/components/module-editor/SideBarModuleOverview.tsx

```diff
@@ -59,36 +64,12 @@ const SideBarModuleOverview = (): React.ReactElement => {
         orderedLessons.map((lesson, index) =>
           state.lessons[focusedLesson] === lesson ? (
             lesson && (
-              <Button
-                key={index}
-                onClick={() => setFocus(index)}
-                variant="unstyled"
-                borderLeftColor="brand.royal"
-                borderLeftWidth="5px"
-                borderRadius="0"
-                bg="background.light"
-                textAlign="left"
-                pl="30px"
-                minH="55px"
-                w="100%"
-              >
-                {lesson.title}
-              </Button>
+              <LessonOptions text={lesson.title} isFocused isHovered={isHovered} index={index} setIsHovered={setIsHovered} setFocus={setFocus}/>                  
```

Comment:
We should pass `state.lessons[focusedLesson] === lesson` as the value of `isFocused` so that we can remove the duplicated `<LessonOptions>` line.